install.sh

```diff
@@ -155,6 +155,10 @@ echo "==== Create $env_name conda environment"
 
 conda env create --name "$env_name" --force --file="$conda_env_file"
 
+# Avoid a 'frankenconda' env that uses Python from the base env.
+# See https://github.com/illinois-ceesd/emirge/pull/132 for details.
+conda deactivate
+conda deactivate
```

Comment:
Wouldn't the second one tend to fail on "normal" machines?

Reply:
No, `conda deactivate` will always succeed as long as `conda` is available.

Reply:
You're right. Yikes. TIL.